benchexec/tools/gazer-theta.py

```diff
@@ -20,7 +20,10 @@ class Tool(benchexec.tools.template.BaseTool2):
     REQUIRED_PATHS = [".."]
 
     def executable(self, tool_locator):
-        return tool_locator.find_executable("gazer_starter.py", subdir="scripts")
+        try:
+            return tool_locator.find_executable("gazer-start.sh")
+        except ToolNotFoundException:
```

Comment:
This needs to be imported. Then the PR is ready to merge.

Reply:
True, I forgot to add the include. Fixed in a26ad64.